ethereum/core/src/jmh/java/tech/pegasys/pantheon/ethereum/vm/operations/BlockHashOperationBenchmark.java

```diff
@@ -42,6 +42,7 @@
   @Setup
   public void prepare() throws Exception {
     operationBenchmarkHelper = OperationBenchmarkHelper.create();
+    // ?? do we create another benchmark ??
```

Comment:
No, I think the Constantinople benchmark is all we need here as the operation of block hash is unaffected (technically we could use a much earlier milestone) and the gas calculator isn't what we're trying to benchmark anyway.

Reply:
Now that I added a PetersburgGasCalculator I moved it over to that one, just to keep current.